src/mongocxx/pipeline.hpp

```diff
@@ -120,6 +120,16 @@ class MONGOCXX_API pipeline {
     pipeline& redact(bsoncxx::document::view_or_value restrictions);
 
     ///
+    /// Randomly selects the specified number of documents that pass into the stage and passes the
+    /// remaining documents to the next stage in the pipeline.
+    ///
+    /// @see http://docs.mongodb.org/manual/reference/operator/aggregation/sample/#pipe._S_sample
+    ///
+    /// @param set the number of input documents to select.
+    ///
+    pipeline& sample(std::int32_t size);
```

Comment:
Why int32_t here, rather than, say, size_t? Does the $sample operator specify a width for this parameter?